ts/src/base/Exchange.ts

```diff
@@ -3963,16 +3962,52 @@ export default class Exchange {
         if (vwap === undefined) {
             vwap = Precise.stringDiv (this.omitZero (quoteVolume), baseVolume);
         }
-        if ((last !== undefined) && (close === undefined)) {
-            close = last;
-        } else if ((last === undefined) && (close !== undefined)) {
-            last = close;
+        // calculate open
+        if (change !== undefined) {
```

Comment:
the thing is, I'm not sure if some exchanges don't return the change as an absolute value like (50) regardless or + 50 or -50 🤔  maybwe we need to check that